### ISSUE TITLE
Add support for widgets

### ISSUE THREAD

**Issue by pravi** [Nheko-Reborn/nheko#82]
### System:

- Nheko version: 0.6.4
- Installation method: .deb
- Operating System: debian buster backports
- Qt version: 
- C++ compiler: 
- Desktop Environment: 

### Actual behavior

No support for adding or viewing etherpad widget.

### Expected behavior

I should be able to view attached etherpad and other widgets.

**Comment by z33ky** [comment]
I don't know how widgets are generally implemented in Matrix. I know for jitsi there is a separate URL available for the meeting room. It'd be nice even to just have a listing of these "active" widgets and their external URL, even if their content isn't displayed (yet).

**Comment by deepbluev7** [comment]
Yeah, we should really display the links. That means at least some widgets can be somewhat used. Many newer widgets require client integration though (like FOSDEM), so those won't work :/

**Comment by rathann** [comment]
FWIW, the YouTube widget does not work, either. Clicking the displayed URL opens Firefox with https://scalar.vector.im/api/widgets/youtube.html?url=https%253A%252F%252Fwww.youtube.com%252Flive%252... (trimmed) and an error message saying "API Error (403): Missing access token".
